certified-connectors/Secib/apiDefinition.swagger.json

```diff
@@ -0,0 +1,755 @@
+                          "type": "string",
+                          "description": "Fullname of the case file's lead attorney.",
+                          "title": "Lead attorney's fullname"
+                        },
+                        "actif": {
+                          "type": "boolean",
+                          "description": "Is the lead attorney active ?",
+                          "title": "Active state",
+                          "enum": [true, false]
+                        },
+                        "email": {
+                          "type": "string",
+                          "description": "Email of the case file's lead attorney.",
+                          "title": "Lead attorney's email",
+                          "x-ms-visibility": "important"
+                        },
+                        "login": {
+                          "type": "string",
+                          "description": "Login of the case file's lead attorney.",
+                          "title": "Lead attorney's login"
+                        }
+                      },
+                      "description": "Lead attorney of the case file"
+                    }
+                  },
+                  "description": "result"
+                }
+              }
+            }
+          },
+          "310": {
+            "description": "Too Many Redirects",
+            "schema": {}
+          },
+          "400": {
+            "description": "Bad Request",
+            "schema": {}
+          },
+          "401": {
+            "description": "Unauthorized request",
+            "schema": {},
+            "headers": {
+              "WWW-Authenticate": {
+                "description": "WWW-Authenticate",
+                "type": "string"
+              }
+            }
+          },
+          "403": {
+            "description": "Forbidden",
+            "schema": {}
+          },
+          "404": {
+            "description": "Not Found",
+            "schema": {}
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "schema": {}
+          },
+          "503": {
+            "description": "Service Unavailable",
+            "schema": {}
+          }
+        },
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/api/v1/secib/Dossiers/events/created": {
+      "x-ms-notification-content": {
+        "schema": {
+          "type": "object",
+          "properties": {
+            "result": {
+              "type": "object",
+              "properties": {
+                "dossierId": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Identifier of the case file.",
+                  "title": "Identifier"
+                },
+                "code": {
+                  "type": "string",
+                  "description": "Number of the case file.",
+                  "title": "Case file n\u00b0",
```

Comment:
@thomasBeckSecib Unicodes - The connector's swagger still have unicodes which may not be supported in all the Power Automate page, Can you please replace all the unicodes with respective characters in the English language.

Reply:
Every unicodes characters are now removed from artifacts.